### PR TITLE
Create PR template for data modeling updates

### PR DESCRIPTION
**Pull request by ayrenay**
## About
I'm proposing a pull request template for data modeling PRs. I believe will make for more consistent PR reviews internally and with our clients.

### CODE REVIEW

**Comment by ayrenay** [comment]
@RAbramson1 + @sean-rose — I just made an actual PR so that we can iterate through these changes a bit easier.

At last discussion, @sean-rose, you mentioned using more semantic markup but suggested h3. I opted for h1 in the spirit of being closer to the semantic intentions.

File: template-data-modeling-pr.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- _List open questions and other things your reviewer should know_
+
+# Related links
+- _List links to related ticket(s)_
+- _(optional) Link to other files and resources, such as specific related reports in a BI tool_
+
+# Checklist
+- [ ] All models, whether directly changed by this model or not, run successfully
+- [ ] New code follows [dbt coding conventions](https://github.com/brooklyn-data/co/blob/master/dbt_coding_conventions.md) and [SQL style guide](https://github.com/brooklyn-data/co/blob/master/sql_style_guide.md)
+- [ ] Testing
```
Comment (sean-rose) [inline template-data-modeling-pr.md:17]:
```suggestion
- [ ] Testing:
```

File: template-data-modeling-pr.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# Other notes
+- _List open questions and other things your reviewer should know_
+
+# Related links
+- _List links to related ticket(s)_
+- _(optional) Link to other files and resources, such as specific related reports in a BI tool_
+
+# Checklist
+- [ ] All models, whether directly changed by this model or not, run successfully
```
[the remark this answers is not in the document]
Comment (ayrenay) [reply template-data-modeling-pr.md:15]: My personal preference is no periods 😄  

I did a quick Google search though and, for complete sentences (as opposed to something like a list of words), it looks like grammatical rules suggest you should have periods, so I will update!

**Comment by ayrenay** [comment]
> A couple of recommended changes, and one suggestion.
> 
> Also, a couple of things about the file name:
> 
> * The existing files in this repo use underscores to separate words rather than dashes, so I'd recommend using underscores here too.
> * I think a slightly different file name like `pr_template_data_modeling.md` could be preferable so that all PR templates would be grouped together.  Or we could have a subfolder for them, like `pr_templates/data_modeling_pr.md`.

Makes sense on the underscores, and I am not a huge fan of the file name as I wrote it. I'm more inclined for a subfolder because I think that's more readable. Any reason why you wouldn't, @sean-rose ?

**Comment by sean-rose** [comment]
> I'm more inclined for a subfolder because I think that's more readable. Any reason why you wouldn't, @sean-rose ?

I'm in favor of having a subfolder. 👍

File: template-data-modeling-pr.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Overview
```
[the remark this answers is not in the document]
Comment (ayrenay) [reply template-data-modeling-pr.md:1]: Something feels weird about the 'Overview' being the "Most important heading" as identified by an `h1`, but I can't think of another alternative that makes sense in the context of actually using this in a PR, which would already have its own title. What do you think of them all being `h2` then?

[the remark this answers is not in the document]
Comment (sean-rose) [reply template-data-modeling-pr.md:1]: Yeah, GitHub displays the PR's title as an `h1`, and I was suggesting "Overview" be an `h2` (as shown in the "Suggested change" code diff).

[the remark this answers is not in the document]
Comment (ayrenay) [reply template-data-modeling-pr.md]: Don't know how I missed that but thanks for calling that out!

**Comment by sean-rose** [comment]
One final idea on the file name:  maybe `dbt_modeling_pr.md` since it has some dbt-specific stuff.

**Pull request merged**